app/controllers/admin/overview_controller.rb

```diff
@@ -1,6 +1,11 @@
 module Admin
   class OverviewController < Admin::ApplicationController
     layout "admin"
-    def index; end
+    def index
+      @open_abuse_reports_count =
+        FeedbackMessage
+          .where(status: "Open", feedback_type: "abuse-reports")
```

Comment:
could we make this a scope instead or a class method on Feedback message?

`scope open_abuse_reports`
or
`def self.open_abuse_report_count`

It's might easier to avoid duplication later if we don't keep this logic in the controller. The diff gets a little larger though :wink: